src/core/sys/posix/aio.d

```diff
@@ -0,0 +1,74 @@
+/**
+ * D header file to interface with the
+ * $(HTTP pubs.opengroup.org/onlinepubs/9699919799/basedefs/aio.h.html, Linux aio API).
```

Comment:
I'd still say `Posix AIO` here, as I've repeatedly said, this is _not_ a linux module. ;-)